types/node/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for non-npm package Node.js 12.11
+// Type definitions for non-npm package Node.js 12.12
```

Comment:
should we use 12.13 here?

Reply:
12.13 didn't really introduce anything new.

Reply:
So, let's use 12.13 here. Could we?

Reply:
There is no advantage in doing so and would just trigger another build and delay things further, I will leave as is *unless* another change will be made.